### ISSUE TITLE
leading slash directory doesn't work. you will see all files within the root directory

### ISSUE THREAD

**Issue by rlam3** [mherrmann/gitignore_parser#65]
hey @mherrmann. thanks for this python module.

we discovered that a .gitignore with a leading slash directory doesn't work
you will see all files within the root directory

for instance 

```
root
--node_modules
--etc 
```

whenever you read a .gitignore with a leading slash it still is not able to pick up those paths

example .gitignore may look like the following

```
/node_modules
```

and this will return false coming back

**Comment by Illegal-Services** [comment]
UP

**Comment by astos-marcb** [comment]
Thanks @rlam3 for bringing that _specific_ one to [my attention](https://gist.github.com/astos-marcb/a673d80b65b6a443559277fb861f9036/revisions#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947).

@pacha it seems `py_walk v0.3.1` is still failing on this regarding the anchored condition part.

**Comment by pacha** [comment]
@astos-marcb You're correct. I fixed it now for `py_walk v0.3.2`. Thanks a lot for letting me know!
